SecretSanta/test/SecretSanta.Api.Tests/Controllers/GroupControllerTests.cs

```diff
@@ -49,36 +50,36 @@ public void GetAllGroups_ReturnsGroups()
         }
 
         [TestMethod]
-        public void CreateGroup_RequiresGroup()
+        public async Task CreateGroup_RequiresGroup()
         {
             var service = new Mock<IGroupService>(MockBehavior.Strict);
             var controller = new GroupsController(service.Object, Mapper.Instance);
 
 
-            var result = controller.Post(null) as BadRequestResult;
+            var result = await controller.Post(null) as BadRequestResult;
 
             Assert.IsNotNull(result);
         }
 
         [TestMethod]
-        public void CreateGroup_ReturnsCreatedGroup()
+        public async Task CreateGroup_ReturnsCreatedGroup()
         {
             var group = new GroupInputViewModel
             {
                 Name = "Group"
             };
             var service = new Mock<IGroupService>();
             service.Setup(x => x.AddGroup(It.Is<Group>(g => g.Name == group.Name)))
-                .Returns(new Group
+                .Returns(Task.FromResult(new Group
```

Comment:
This works, but there is also a convenient helper method that will save you from needing to type `Task.FromResult`
```suggestion
                .ReturnsAsync(new Group
```